lib/generators/rolemodel/all_generator.rb

```diff
@@ -3,18 +3,19 @@ class AllGenerator < Rails::Generators::Base
     source_root File.expand_path('templates', __dir__)
 
     def run_all_the_generators
+      generate 'rolemodel:css:all'
```

Comment:
What's the reasoning for changing the order of all these generators?

Reply:
@Jeremy-Walton, I just like how alphabetic order looks. 
BTW, can you tell me if the running order of generators matters here?

Reply:
It does. Some generators depend on other ones being run first. I.E. CSS depends on icons I think.

All that being said, RoleModel Rails is hugely out of date. The CSS does not reflect our latest learnings (doesn't pull in the design system), and it definitely does not support Rails 7

Reply:
Ah, I see. Thank you for pointing me out!

Reply:
Resolved.